wrench/src/reftest.rs

```diff
@@ -205,25 +241,38 @@ pub fn run_reftests(wrench: &mut Wrench, window: &mut WindowWrapper, filename: &
 
                 false
             },
-            (ReftestOp::NotEqual, ReftestImageComparison::Equal) => {
+            (&ReftestOp::NotEqual, ReftestImageComparison::Equal) => {
                 println!("REFTEST TEST-UNEXPECTED-FAIL | {} | image comparison", name);
                 println!("REFTEST TEST-END | {}", name);
 
                 false
             },
-            (ReftestOp::NotEqual, ReftestImageComparison::NotEqual{..}) => true,
+            (&ReftestOp::NotEqual, ReftestImageComparison::NotEqual{..}) => true,
         };
 
-        if success {
-            total_passing += 1;
-        } else {
-            total_failing += 1;
-        }
-    });
-
-    println!("REFTEST INFO | {} passing, {} failing", total_passing, total_failing);
+        success
```

Comment:
no need to have the `success` variable now at all, we can just end the function with that `match` statement